### PR TITLE
Fixed typos and added suggestions to the text

### PR DESCRIPTION
**Pull request by arianesasso**
This is my first contribution, so I tried to fix some typos and made minor suggestions to the text to improve readability.

### CODE REVIEW

File: .github/workflows/book.yml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 name: deploy-book
 
-# Run on pushes
-on:
-  push:
-    branches:
-      - '*'
+# Run on pushes & pull requests
+on: [push, pull_request]
```
Comment (lwasser) [inline .github/workflows/book.yml:4]: @kysolvik made this change in another PR so i just added it so we could build the docs here and see that all is good with sphinx!

File: open-source-software-submissions/editors-guide.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 Thank you for your time in serving as an editor for a PyOpenSci package! An editor
 for pyOpenSci should have the following experience:
 
-* They should have completed a review for *atleast* 1 package for pyOpenSci.
-* They should have some experience with the open source software for scientific
-Python community be it maintaining or contributing to packages, or being involved
+* They should have completed a review for *at least* 1 package for pyOpenSci.
+* They should have some experience with open source software for the scientific
+Python community, be it maintaining or contributing to packages or being involved
 in the community in some way.
```
Comment (lwasser) [inline open-source-software-submissions/editors-guide.md:9]:
```suggestion
* They should have some experience with open source software that supports the scientific
Python community, be it maintaining or contributing to packages or being involved
in the community in some way.
```

File: open-source-software-submissions/editors-guide.md [hunk 3]
```diff
@@ -18,18 +18,18 @@ pyOpenSci for review.
 Once you begin the review process as an editor:
 
 * Tag the submission issue with the `2/seeking-reviewer(s)` tag.
-* Check the comment submission by the author to ensure that mandatory parts of submission template are complete.
-  - If elements are not complete, direct authors toward filling in any missing pieces.
+* Check the submission by the author to ensure that mandatory parts of the template are complete.
```
Comment (lwasser) [inline open-source-software-submissions/editors-guide.md:21]: much better :)

File: open-source-software-submissions/editors-guide.md [hunk 4]
```diff
@@ -100,15 +100,15 @@ moving.
...
+* Other [contributors to pyOpenSci](https://www.pyopensci.org/contributors/).
 
 When these sources of information are not enough:
-* ping other editors for ideas.
-* look for users of the package or of the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
+* Ping other editors for ideas.
+* Look for users of the package or the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
 * You can also search for authors/maintainers of related packages on [PyPI](https://pypi.org/search/).
-* Post in twitter and ensure pyOpenSci retweets your post.
+* Post in Twitter and ensure pyOpenSci retweets your post.
```
Comment (lwasser) [inline open-source-software-submissions/editors-guide.md:111]:
```suggestion
* Post on Twitter and ensure pyOpenSci retweets your post.
```

**Comment by lwasser** [comment]
@arianesasso this is great! i think we can merge this once ci builds! i am digging as i thought there were some other comments that you made about moving things around.. so will look for those next.

**Comment by lwasser** [comment]
@arianesasso i swear you had a few suggestions  that weren't implemented yet... maybe they were on slack... will go there next.

**Comment by arianesasso** [comment]
Thanks @lwasser! Yes, some of the suggestions are still on slack because they were a bit "bigger" haha. But I can open another pull request :).

**Comment by lwasser** [comment]
ok so @arianesasso this is one reason to pull down a pr and build hte book locally. i'm getting a super funky build here. 

You can actually see it if you look at the CI CHECK ABOVE - "Click to preview rendered book"
or just click below. It looks like the grid directive is not working but I know you didn't actually edit that page. 

https://output.circle-artifacts.com/output/job/02cb0329-fff0-4a70-8112-7e36df71c783/artifacts/0/html/intro.html 

i'm going to see if i can figure out what is going on. Have you by chance updated your branch from main / rebased lately? i wonder if it has something to do with that?

**Comment by lwasser** [comment]
I made another small change - i suspect that will fix things. I am not sure why all of a sudden that is causing issues but i removed the copybutton extension.

**Comment by lwasser** [comment]
ok merging this! thank you @arianesasso the broken things are largely fixed on main and i'm still working on random broken links. A rebase might have helped with some of the issues here. but i am actually not sure why that extension started to cause issues.

**Comment by arianesasso** [comment]
Thank you for reviewing this PR :). Sorry for the confusion, maybe I should have checked the main first?

**Pull request merged**